pkg/extensions/search/schema.graphql

```diff
@@ -109,19 +133,53 @@ type LayerHistory {
     HistoryDescription: HistoryDescription
 }
 
+"""
+Contains details about the supported OS and architecture of the image
+"""
 type OsArch {
     Os: String
     Arch: String
 }
 
 type Query {
+    """
+    Returns a CVE list for the image specified in the arugment
+    """
     CVEListForImage(image: String!): CVEResultForImage!
+
+    """
+    Returns a list of images vulnerable to the CVE of the specified ID
+    """
     ImageListForCVE(id: String!): [ImageSummary!]
+
+    """
+    Returns a list of images that are no longer vulnerable to the CVE of the specified ID, from the specified image (repo)
+    """
     ImageListWithCVEFixed(id: String!, image: String!): [ImageSummary!]
+
+    """
+    Returns a list of images which contain the specified digest 
+    """
     ImageListForDigest(id: String!): [ImageSummary!]
+
+    """
+    Returns a list of repos with the newest tag within
+    """
     RepoListWithNewestImage: [RepoSummary!]!  # Newest based on created timestamp
+
+    """
+    Returns all the images from the specified repo
+    """
     ImageList(repo: String!): [ImageSummary!]
+
+    """
+    Returns information about the specified repo
+    """
     ExpandedRepoInfo(repo: String!): RepoInfo!
+
+    """
+    Searches within repos, images, and layers
+    """
     GlobalSearch(query: String!): GlobalSearchResult!
     DerivedImageList(image: String!): [ImageSummary!]
```

Comment:
Can you also add documentation for the last queries added in the meantime?

Reply:
Added